src/aui/framemanager.cpp

```diff
@@ -2532,10 +2544,22 @@ void wxAuiManager::Update()
             {
                 // frame already exists, make sure its position
                 // and size reflect the information in wxAuiPaneInfo
-                if ((p.frame->GetPosition() != p.floating_pos) || (p.frame->GetSize() != p.floating_size))
+                // give floating_client_size precedence over floating_size
```

Comment:
But we still end up calling `SetSize()`? I thought the whole idea was to call `SetClientSize()`.

Reply:
The code uses `ClientToWindowSize()` to convert `floating_client_size` to a window size, and then calls `SetSize()`.  The reason to use `SetSize()` is that it can set both position and size, but I don't see a way for `SetClientSize()` to set the window position.  (My reading of the `SetClientSize(const wxRect&)` implementation is that it ignores the position!)  Are you suggesting rewriting the code to use separate `SetPosition()` and `SetClientSize()` calls?

Reply:
Done.  See 15c3a445fbeca1513ab0fe7851ffba746fbcb309